src/components/ProgressCircular/ProgressCircularSVG.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path } from 'react-native-svg';
+import {
+    COLOR_BRAND,
+    COLOR_SUCCESS,
+    COLOR_ERROR,
+    COLOR_WARNING,
+} from '../../styles/colors';
+import { BaseProps } from '../types';
+
+const colors = {
+    brand: COLOR_BRAND,
+    success: COLOR_SUCCESS,
+    warning: COLOR_WARNING,
+    error: COLOR_ERROR,
+};
+
+type Variants = 'brand' | 'success' | 'warning' | 'error';
+
+interface Props extends BaseProps {
+    variant?: Variants;
+    value?: number;
+}
+
+/**
+ * A profress circular svg component represents an object or entity
+ */
```

Comment:
Descriptions only need to be in index.tsx. This descriptions are for future documentation of the component and its props.